dcm/src/main/java/org/dcm/viewupdater/ViewUpdater.java

```diff
@@ -0,0 +1,215 @@
+                        break;
+                    case STRING_TYPE:
+                        objects.add(f.getString());
+                        break;
+                    default:
+                        throw new RuntimeException("Unexpected datatype: " + cls.getName());
+                }
+                counter = counter + 1;
+            }
+            recordsFromDDLog.computeIfAbsent(record.getStructName(), k -> new ArrayList<LocalDDlogCommand>());
+            recordsFromDDLog.get(tableName).add(new LocalDDlogCommand(command.kind().toString(), tableName, objects));
+        }
+    }
+
+    public void flushUpdates() {
+        updater.sendUpdatesToDDlog(mapRecordsFromDB.get(modelName));
+
+        for (final Map.Entry<String, List<LocalDDlogCommand>> entry: recordsFromDDLog.entrySet()) {
+            final String tableName = entry.getKey();
+            final List<LocalDDlogCommand> commands = entry.getValue();
+                for (final LocalDDlogCommand command : commands) {
+                    // check if query is already created and if not, create it
+                    if (!preparedQueries.containsKey(tableName) ||
+                            (preparedQueries.containsKey(tableName) &&
+                                    !preparedQueries.get(tableName).containsKey(command.command))) {
+                        updatePreparedQueries(tableName, command);
+                    }
+                    flush(tableName, command);
+                }
+        }
+        recordsFromDDLog.clear();
+        mapRecordsFromDB.get(modelName).clear();
+    }
+
+     private void flush(final String tableName, final LocalDDlogCommand command) {
+        try {
+            final PreparedStatement query = preparedQueries.get(tableName).get(command.command);
+            final IRTable irTable = irTables.get(tableName);
+            final Table<? extends Record> table = irTable.getTable();
+            final Field[] fields = table.fields();
+            for (int i = 0; i < fields.length; i++) {
+                final Class fieldClass = fields[i].getType();
+                final Object item = command.values.get(i);
+                final int index = i + 1;
+                switch (fieldClass.getName()) {
+                    case LONG_TYPE:
+                        if (item.getClass().getName().equals(BIGINT_TYPE)) {
+                            query.setInt(index, ((java.math.BigInteger) item).intValue());
+                        } else {
+                            query.setLong(index, (Long) item);
+                        }
+                        break;
+                    case INTEGER_TYPE:
+                        query.setInt(index, (Integer) item);
+                        break;
+                    case BOOLEAN_TYPE:
+                        query.setBoolean(index, (Boolean) item);
+                        break;
+                    default:
+                        query.setString(index, (String) item);
```

Comment:
In scenarios where you expect more cases to be added in the future, always handle known cases explicitly, and throw an exception from the default case. So add a case for strings.